changelog/23580.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ui: Fix namespace picker and console when in collapsed navbar view.
```

Comment:
I'd maybe mention upgrading HDS here to fix sidebar navigation issues when collapsed